tests/commands/autoupdate_test.py

```diff
@@ -474,3 +474,23 @@ def test_updates_old_format_to_new_format(tmpdir, capsys, store):
     )
     out, _ = capsys.readouterr()
     assert out == 'Configuration has been migrated.\n'
+
+
+def test_maintains_rev_quoting_style(tmpdir, out_of_date, store):
+    fmt = (
+        'repos:\n'
+        '-   repo: {0}\n'
+        '    rev: "{1}"\n'
+        '    hooks:\n'
+        '    -   id: foo\n'
+        '-   repo: {0}\n'
+        '    rev: \'{1}\'\n'
```

Comment:
can avoid escapes by doing this:
```suggestion
        "    rev: '{1}'\n"
```